### DISCUSSION TITLE
Posttranslational modifications

### DISCUSSION THREAD

**Discussion by martinbonin** [chaidiscovery/chai-lab#83] [Q&A] [answered]
Hi, can chai-1 handle posttranslational modifications such as phosphorylations or glycosylations? If so, how can I input these using the webserver?

**Comment by arogozhnikov** [comment, marked as answer]
Hi @martinbonin 

PTMs not supported yet, we need to figure out how to properly include this in FASTA, see this comment: https://github.com/chaidiscovery/chai-lab/issues/56#issuecomment-2358533905

recommendations and references to existing approaches are welcome.

**Reply by arogozhnikov** [reply]
Update, we have now example of input here: https://github.com/chaidiscovery/chai-lab/tree/main/examples/covalent_bonds
